mocked_servers/https/Dockerfile

```diff
@@ -1,11 +1,9 @@
-FROM node:14
-
-WORKDIR /app
-
-COPY certificates ./certificates
-COPY *.js ./
-COPY package.json ./
-
-RUN npm install
-
-CMD ["node", "index.js"]
+FROM golang:1.17 as build
+WORKDIR $GOPATH/main
+COPY . .
```

Comment:
Is the entire directory needed or do you only need to copy certain files?

Reply:
I would assume yes on this.

Reply:
There's only 4 files in the directory to begin with. I guess I don't need to pull in the Dockerfile, but I don't think that'll make much of a difference.